### PR TITLE
Added the merged markdown content for the Pre-Reading-Material

### PR DESCRIPTION
**Pull request by prabhukiran9999**
Content can be viewed at https://prabhukiran9999.github.io/cloud-pathfinder/

### CODE REVIEW

**Review by wrnu — CHANGES_REQUESTED** [review]
@prabhukiran9999 Could you please set the PR title to something descriptive and add a summary of the PR contents.

**Comment by prabhukiran9999** [comment]
> @prabhukiran9999 Could you please set the PR title to something descriptive and add a summary of the PR contents.

Sorry, did that :blush:

**Review by wrnu — APPROVED** [review]

File: vuepress/src/Documentation/Pre-reading-material/000200_sea_at_bc_Gov.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+- **Lambdas** = a serverless technology where code is run on CSP maintained servers, turned off when not in use
+- **IAM** = Identity Access Management of users in the CSP
+- **AI/ML** = Artificial Intelligence and Machine Learning Cloud services 
+  :::
+
+  * GCP
+
+![compliant services in GCP](./images/compliant_services_gcp.png)
+::: tip Acronyms
+- **VMs:** Elastic Compute Cloud Virtual Machines
```
Comment (ActionAnalytics) [inline vuepress/src/Documentation/Pre-reading-material/000200_sea_at_bc_Gov.md:37]: I think the GCP VMs likely have a different brand name

File: vuepress/src/Documentation/Pre-reading-material/000600_billing.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+| Data Transfer Out   | 5000 GB      | $ 450.00  |
+| Amazon CloudWatch Logs   | 5000 GB      | $ 2,775.00  |
+| AWS SEA Services(*)   | Amazon GuardDuty, AWS Config, KMS      | $ 40.00  |
+| Estimated Monthly Total   | --     | $ 4,500.16  |
+ * ALB: 10.8gb/hr, 250connections/sec, 2min duration,10 request per second
+ * Factor in backups, cold storage, dev & test environments which will increase costs depending on how far a team chooses to implement.
+## Custom Cost Examples ?
+ * Yes!
+ ![Screen Shot of the AWS Price Calculator page](./images/aws_cost_calculator.png)
+ * Use [AWS Pricing Calculator](https://calculator.aws) for now
```
Comment (ActionAnalytics) [inline vuepress/src/Documentation/Pre-reading-material/000600_billing.md:61]: Let's link to the Azure and GCP cost estimators too

File: vuepress/src/Documentation/Pre-reading-material/000700_communities_of_practise.md [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+- **CoP** = Community of Practise
+
+- **SecOps** = OCIO Security Operations Team
+
+:::
+
+---
+* Our Customers: Project Managers / Product Owners / Expense Authorities  
+* Our Users: Developer Teams
+* Special Users: Security Operations OCIO
```
Comment (ActionAnalytics) [inline vuepress/src/Documentation/Pre-reading-material/000700_communities_of_practise.md:29]: Updating as we go, let's rename "Special Users" to "Compliance Users".  After Security Operations OCIO, MISOs

File: vuepress/src/Documentation/Pre-reading-material/000700_communities_of_practise.md [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+## Customer Focus – 3 CoPs
+---
+::: tip Acronyms
+
+- **OCIO** = Office of the Associate Deputy Minister and Government Chief Information Officer
+
+- **CoP** = Community of Practise
+
+- **SecOps** = OCIO Security Operations Team
+
```
Comment (ActionAnalytics) [inline vuepress/src/Documentation/Pre-reading-material/000700_communities_of_practise.md:23]: **MISO** = Ministry Information Security Officer, embedded as part of the developer team

**Review by ActionAnalytics — CHANGES_REQUESTED** [review]
Looks great, just a couple minor tweaks

File: vuepress/src/Documentation/Pre-reading-material/000200_sea_at_bc_Gov.md (hunk 1 above)
Comment (prabhukiran9999) [reply vuepress/src/Documentation/Pre-reading-material/000200_sea_at_bc_Gov.md]: Yep, Sorry it should be Google Compute Engines.

**Review by ActionAnalytics — APPROVED** [review]

**Pull request merged**